packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -2192,35 +2248,56 @@ class AnnotatedRegionLayer<T> extends ContainerLayer {
   /// If not provided, offset defaults to [Offset.zero].
   ///
   /// Ignored if [size] is not set.
+  ///
+  /// Unlike [OffsetLayer], this parameter [offset] does not affect the layer's
+  /// children.
   final Offset offset;
 
-  @override
-  S find<S>(Offset regionOffset) {
-    final S result = super.find<S>(regionOffset);
-    if (result != null)
-      return result;
-    if (size != null && !(offset & size).contains(regionOffset))
-      return null;
-    if (T == S) {
-      final Object untypedResult = value;
-      final S typedResult = untypedResult;
-      return typedResult;
-    }
-    return null;
-  }
+  /// Whether this layer should prevent layers visually behind it from being
+  /// hit during a hit test.
+  ///
+  /// If [opaque] is true, this layer will absorb the hit if this layer contains
+  /// the target position and has the requested annotation.
+  ///
+  /// If [opaque] is false, this layer will not change the absorption value
+  /// returned by its children.
+  ///
+  /// This defaults to false.
+  ///
+  /// See also:
+  ///
+  ///  * [HitTestBehavior], which controls similar logic when hit-testing in the
+  ///    render tree.
+  final bool opaque;
```

Comment:
You should describe how this is affected by the `onlyFirst` argument in `findAnnotations`.